docs/Manuals/Program/freeciv21-client.rst

```diff
@@ -0,0 +1,151 @@
+===========
+
+.. include:: freeciv21-server.rst
+  :start-line: 17
+  :end-line: 29
+
+This is the client interface `program` used to connect to a Freeciv21 server. For more information on the
+server, refer to freeciv21-server(6). The program can be launced without any command line parameters. A player
+can use the main menu to connect to a running game, open an existing savegame, start a new game, or start a
+new scenario game. However, if you wish to automate the connection to a running server or start a new game
+with a server settings file, command line options are available.
+
+OPTIONS
+=======
+
+The following options are accepted on the command line of the client. They may not be combined as with other
+tools. For example: ``freeciv21-client -sp localhost 5556`` will not work. Instead you will need to enter
+each option separately, such as, ``freevic21-client -s localhost -p 5556``.
+
+``-A, --Announce <PROTO>``
+    Announce game in LAN using protocol PROTO (IPv4/IPv6/none), Default is ``IPv4``. Options include:
+    ``IPv4``, ``IPv6``, or ``none``.
+
+``-a, --autoconnect``
+    Skip connection dialog (usually with `url` or ``-n``, ``-s``, and ``-p``).
+
+``-d, --debug <LEVEL>``
+    Set debug log level (fatal/critical/warning/info/debug). Default log level is ``info``.
+
+``-F, --Fatal``
+    Raise a signal on failed assertion. An assertion is a code calculation error. With this set, the client
+    process will SEGFAULT instead of issuing a warning message to the terminal console.
+
+``-f, --file <FILE>``
+    Load saved game FILE. Useful when wanting to restart a game.
+
+``-H, --Hackless``
+    Do not request hack access to local, but not spawned, server.
+
+``-l, --log <FILE>``
+    Use FILE as logfile.
+
+``-M, --Meta <HOST>``
+    Connect to the metaserver at HOST.
+
+.. note::
+  With regard to the command line arguments concerning the metaserver, Freeciv21 does not have its own
+  metaserver at this time and the legacy Freeciv metaserver does not support Freeciv21. This means, right now,
+  that all commands related to the metaserver are held over from the fork from legacy Freeciv until the
+  Longturn community creates a custom metaserver.
+
+
+``-n, --name <NAME>``
+    Use NAME as username on server.
+
+``-p, --port <PORT>``
+    Connect to server port PORT (usually with ``-a``).
+
+``-P, --Plugin <PLUGIN>``
+    Use PLUGIN for sound output []. The default is `SDL`.
+
+``-r, --read <FILE>``
+    Read startup script FILE. Options are passed to the spawned server.
+
+``-s, --server <HOST>``
+    Connect to the server at HOST (usually with ``-a``).
+
+``-S, --Sound <FILE>``
+    Read sound tags from FILE. Default is `stdsounds.soundspec`.
+
+``-m, --music <FILE>``
+    Read music tags from FILE. Default is `stdmusic.musicspec`.
+
+``-t, --tiles <FILE>``
+    Use data file `FILE.tilespec` for tiles.
+
+``-w, --warnings``
+    Warn about deprecated modpack constructs.
+
+``-h, --help``
+    Displays help on command line options.
```

Comment:
Should this and the following 3 use "Display" instead of "Displays"? The other options seem to use the imperative form.

Reply:
That is the client helptext output. Might look to change the code too.

Reply:
Turns out this comes from Qt, so can't change in code. Will fix in man page.